scripts/GetClrDbg.ps1

```diff
@@ -0,0 +1,118 @@
+﻿<#
+.SYNOPSIS
+Downloads the given $Version of clrdbg for the given $RuntimeID and installs it to the given $InstallPath
+
+.DESCRIPTION
+The following script will generate a project.json and NuGet.config and use dotnet resotre and publish to install clrdbg, the .NET Core Debugger
+
+.PARAMETER Version
+Specifies the version of clrdbg to install. Can be 'latest', VS2015U2, or a specific version string i.e. 14.0.25229-preview-2963841
+
+.PARAMETER RuntimeID
+Specifies the .NET Runtime ID of the clrdbg that will be downloaded. Example: ubuntu.14.04-x64. Defaults to the Runtime ID of the current machine.
+
+.Parameter InstallPath
+Specifies the path where clrdbg will be installed. Defaults to the directory containing this script.
+
+.INPUTS
+None. You cannot pipe inputs to GetClrDbg.
+
+.EXAMPLE
+C:\PS> .\GetClrDbg.ps1 -Version latest -RuntimeID ubunut.14.04-x64 -InstallPath .\clrdbg
+
+.LINK
+https://github.com/Microsoft/MIEngine
+#>
+
+Param (
+    [Parameter(Mandatory=$true)]
+    [string]$Version,
+    [string]$RuntimeID,
+    [string]$InstallPath = (Split-Path -Path $MyInvocation.MyCommand.Definition)
+)
+
+function GetDotNetRuntimeID() {
+    $ridLine = dotnet --info | findstr "RID"
```

Comment:
I think you either need to explicitly set the error mode to abort the script, or check for errors, because by default this code will happily continue on if dotnet cannot be found, or if it doesn't have a 'RID' line.